inference-engine/src/inference_engine/include/openvino/runtime/common.hpp

```diff
@@ -19,5 +20,52 @@ namespace runtime {
  * @brief This type of map is commonly used to pass set of parameters
  */
 using ConfigMap = std::map<std::string, std::string>;
+
+/**
+ * @struct ProfilingInfo
+ * @brief Represents basic inference profiling information per node.
+ *
+ * If the node is executed using tiling, the sum time per each tile is indicated as the total execution time.
+ * Due to parallel execution, the total execution time for all nodes might be greater than the total inference time.
+ */
+struct ProfilingInfo {
```

Comment:
Can we have a separate file for this structure?

Reply:
Done